nx.json

```diff
@@ -1,84 +1,85 @@
 {
 	"$schema": "./node_modules/nx/schemas/nx-schema.json",
 	"cli": {
+		"analytics": false,
```

Comment:
Note that changing anything in the json file causes the linter to want to sort it :)